packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -139,7 +139,13 @@ export const createDelegationTracker = ({
     transactionsTracker,
     rewardAccountAddresses$,
     slotEpochCalc$,
-    [...Cardano.RegAndDeregCertificateTypes, ...Cardano.StakeDelegationCertificateTypes]
+    [
+      ...new Set([
+        ...Cardano.RegAndDeregCertificateTypes,
+        ...Cardano.StakeDelegationCertificateTypes,
+        ...Cardano.VoteDelegationCredentialCertificateTypes
```

Comment:
In general `fix:` should have some test (or a change in existing test) and this looks like something testable. Let me know if I'm mistaken and it's not practical

Reply:
I agree and spent some time on it. 
What would be needed is a set of unit tests for `createDelegationTracker`, but given the complexity of the dependencies, one was never created.
The bits and pieces, like `certificateTransactionsWithEpochs` are unit tested, but in this case it's just a matter of certificates being filtered on, passed as an array to `certificateTransactionsWithEpochs`.
So I found it impractical and out of scope to add unit tests for `createDelegationTracker` for this small change.